zulip/zulip/__init__.py

```diff
@@ -951,6 +951,10 @@ def remove_reaction(self, reaction_data: Dict[str, Any]) -> Dict[str, Any]:
         )
 
     def get_realm_emoji(self) -> Dict[str, Any]:
+        logger.warning("get_realm_emoji() is deprecated." " Please use get_custom_emoji() instead.")
```

Comment:
Deprecation warnings should be flagged with [`warnings.warn`](https://docs.python.org/3/library/warnings.html#warnings.warn).

```suggestion
        warnings.warn(
            "get_realm_emoji() is deprecated. Please use get_custom_emoji() instead.",
            DeprecationWarning,
            stacklevel=2,
        )
```